vars/setupCommonPipelineEnvironment.groovy

```diff
@@ -65,20 +88,53 @@ void call(Map parameters = [:]) {
     }
 }
 
-private loadConfigurationFromFile(script, String configFile) {
+private static loadConfigurationFromFile(script, String configFile) {
```

Comment:
Why static?

Reply:
The IDE detects when a method can be static and suggests to make it so. Personally I like when a method is declared "static" as that hints at its "independence" and easy extraction. I am not sure, but I think that is the reason why the IDE suggests to use static when possible.